Compilation errors on mac osx

I'm trying to compile SPX on MacOS (El capitan).  I run into this compilation error : 

I'm just following the documentation, and hit that error at make time. 

```
In file included from /Users/famille/bin/php-spx/src/spx_resource_stats.c:4:
/Users/famille/bin/php-spx/src/spx_resource_stats-macos.c:20:12: error: implicit declaration of function 'spx_resource_stats_wall_time_coarse' is invalid in
      C99 [-Werror,-Wimplicit-function-declaration]
    return spx_resource_stats_wall_time_coarse();
           ^
/Users/famille/bin/php-spx/src/spx_resource_stats-macos.c:20:12: note: did you mean 'spx_resource_stats_wall_time'?
/Users/famille/bin/php-spx/src/spx_resource_stats-macos.c:17:8: note: 'spx_resource_stats_wall_time' declared here
size_t spx_resource_stats_wall_time(void)
       ^
/Users/famille/bin/php-spx/src/spx_resource_stats-macos.c:31:12: error: implicit declaration of function 'spx_resource_stats_cpu_time_coarse' is invalid in
      C99 [-Werror,-Wimplicit-function-declaration]
    return spx_resource_stats_cpu_time_coarse();
           ^
/Users/famille/bin/php-spx/src/spx_resource_stats-macos.c:31:12: note: did you mean 'spx_resource_stats_wall_time_coarse'?
/Users/famille/bin/php-spx/src/spx_resource_stats-macos.c:20:12: note: 'spx_resource_stats_wall_time_coarse' declared here
    return spx_resource_stats_wall_time_coarse();
           ^
/Users/famille/bin/php-spx/src/spx_resource_stats-macos.c:48:22: error: static declaration of 'spx_resource_stats_wall_time_coarse' follows non-static
      declaration
static inline size_t spx_resource_stats_wall_time_coarse(void)
                     ^
/Users/famille/bin/php-spx/src/spx_resource_stats-macos.c:20:12: note: previous implicit declaration is here
    return spx_resource_stats_wall_time_coarse();
           ^
/Users/famille/bin/php-spx/src/spx_resource_stats-macos.c:63:22: error: static declaration of 'spx_resource_stats_cpu_time_coarse' follows non-static
      declaration
static inline size_t spx_resource_stats_cpu_time_coarse(void)
                     ^
/Users/famille/bin/php-spx/src/spx_resource_stats-macos.c:31:12: note: previous implicit declaration is here
    return spx_resource_stats_cpu_time_coarse();
           ^
4 errors generated.

```

More info upon request.

--- a/src/spx_resource_stats-macos.c
+++ b/src/spx_resource_stats-macos.c
@@ -4,6 +4,9 @@
 
 #include "spx_resource_stats.h"
 
+static inline size_t spx_resource_stats_wall_time_coarse(void);
+static inline size_t spx_resource_stats_cpu_time_coarse(void);
+
 void spx_resource_stats_init(void)
 {
 }

